client-course-schedulizer/src/components/pages/HarmonyPage/Harmony/HarmonyAddAssignments.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { HarmonyFormsAccessors, HarmonyFormsState, useHarmonyFormsStore } from "utilities/hooks";
+import { HarmonyCourseCheckboxes } from "./HarmonyCourseChecboxes";
```

Comment:
`./HarmonyCourseChecboxes` should be `./HarmonyCourseCheckboxes`

Reply:
Thanks! If you have more advice on how to lay-out the harmony page, let me know!